config_scheduler.go

```diff
@@ -97,8 +97,8 @@ func (c *SchedulerConfig) ValidateAndDefault() error {
 			ValidHostAllocators, c.HostAllocator)
 	}
 
-	if c.FreeHostFraction < 0 || c.FreeHostFraction > 1 {
-		return errors.New("free host fraction must be between 0 and 1")
+	if c.FutureHostPercent < 0 || c.FutureHostPercent > 100 {
```

Comment:
The old version didn't do this either, but it might be nice to have a default if this is unset